### ISSUE TITLE
v4.1.2 AgendaConfig type issue

### ISSUE THREAD

**Issue by houxinli** [agenda/agenda#1292]
Received update of v4.1.2 which fixes #1265 , thanks!

But the other issue I mentioned was not correctly fixed.

Plus, the AgendaConfig type definition has some problem. In my use case I'm using a mongoDB instanse and the collection property in the db option (not using address and other MongoClientOptions)
```ts
interface AgendaConfig{
    ...
    mongo?: MongoDb;
    db?: {
        address: string;
        collection: string;
        options: MongoClientOptions;
    };
```

_Originally posted by @houxinli in https://github.com/agenda/agenda/issues/1265#issuecomment-790241506_

**Comment by houxinli** [comment]
The db part is currently
```ts
db?: {
    address: string;
    collection?: string;
    options?: MongoClientOptions;
};
```
But it's triggering errors when I'm using mongoDb instance and the collection option. I dived into the package and found that the requested key should be 'collection', isn't it ?
```ts
db?: {
    address?: string;
    collection: string;
    options?: MongoClientOptions;
};
```

**Comment by love8587** [comment]
+1 had a similar issue

**Comment by koresar** [comment]
Could anyone please report if it is fixed in the v4.1.3 ?

**Comment by koresar** [comment]
Closing as fixed. No response from issue starter.
